GitHubComments.ps1

```diff
@@ -62,33 +62,37 @@ function Get-GitHubIssueComment
         [Parameter(Mandatory)]
         [int] $IssueNumber,
 
-        [string] $Since,
+        [DateTime] $Since,
 
         [string] $AccessToken,
 
         [switch] $NoStatus
     )
 
-    Write-InvocationLog -Invocation $MyInvocation
+    Write-InvocationLog
 
-    $elements = Resolve-RepositoryElements -BoundParameters $PSBoundParameters
+    $elements = Resolve-RepositoryElements
     $OwnerName = $elements.ownerName
     $RepositoryName = $elements.repositoryName
 
+    if ($Since -ne $null) {
```

Comment:
PowerShell best-practice is to always compare against `$null` with `$null` on the left, that is, `$null -ne $Since`